bindings/python/test/test_numpy.py

```diff
@@ -154,6 +154,10 @@ def test_write_error(self):
                 raise awe
 
     def test_write_schema_validation(self):
+        try:
+            import pandas as pd  # noqa: F401
+        except ImportError:
+            self.skipTest("Test requires pandas")
         arrow_schema = {k.__name__: v(True) for k, v in _TYPE_NORMALIZER_FACTORY.items()}
         schema = {k: v.to_pandas_dtype() for k, v in arrow_schema.items()}
```

Comment:
We use pandas to go from arrow to numpy?

Reply:
Just for this test, to get the right data types